contractsSol5/Dao/KyberStaking.sol

```diff
@@ -154,6 +153,9 @@ contract KyberStaking is IKyberStaking, EpochUtils, ReentrancyGuard {
     }
 
     event Withdraw(uint curEpoch, address staker, uint amount);
+    // event is fired if something is wrong with withdrawal
+    // even though the withdrawal is still success
```

Comment:
typo "is still success" -->
will still succeed.
or
is still successful

Reply:
Fixed in latest commit